lib/pronto/rubocop/patch_cop.rb

```diff
@@ -22,10 +22,15 @@ def messages
       end
 
       def processed_source
-        @processed_source ||= ::RuboCop::ProcessedSource.from_file(
-          path,
-          rubocop_config.target_ruby_version
-        )
+        @processed_source ||= begin
+          processed_source = ::RuboCop::ProcessedSource.from_file(
+            path,
+            rubocop_config.target_ruby_version
+          )
+          processed_source.registry = registry if processed_source.responds_to(:registry=)
+          processed_source.config = rubocop_config if processed_source.responds_to(:config=)
```

Comment:
```suggestion
          processed_source.registry = registry if processed_source.respond_to?(:registry=)
          processed_source.config = rubocop_config if processed_source.respond_to?(:config=)
```

Reply:
Fixed!

Reply:
I think you made another typo, it's [`respond_to?`](https://ruby-doc.org/core-2.7.6/Object.html#method-i-respond_to-3F) :disappointed:

Reply:
sorry, let me run the tests locally before doing another push :S